programs/bench.c

```diff
@@ -625,10 +642,12 @@ static void BMK_benchFileTable(const char* const * const fileNamesTable, unsigne
         U32 fileNb;
         for (fileNb=0; fileNb<nbFiles; fileNb++) {
             size_t const fileSize = fileSizes[fileNb];
+            BMK_result_t* resulti = (results == NULL) ? NULL : &(results[fileNb]);
```

Comment:
`BMK_result_t* const resulti`

We are basically obsessed by the `const` property in this project.
Try to `const` everything you can.

It's an important hint for the reviewer that the created value will not change.
When applied consistently, its absence means that this value _will_ change,
and therefore, the reviewer must concentrate his attention on it.